appeals/e2e/cypress/e2e/back-office-appeals/issueDecision.spec.js

```diff
@@ -0,0 +1,89 @@
+// @ts-nocheck
+/// <reference types="cypress"/>
+
+import { users } from '../../fixtures/users';
+import { CaseDetailsPage } from '../../page_objects/caseDetailsPage.js';
+import { ListCasesPage } from '../../page_objects/listCasesPage';
+import { DateTimeSection } from '../../page_objects/dateTimeSection';
+import { urlPaths } from '../../support/urlPaths.js';
+import { tag } from '../../support/tag';
+import { happyPathHelper } from '../../support/happyPathHelper.js';
+
+const listCasesPage = new ListCasesPage();
+const dateTimeSection = new DateTimeSection();
+const caseDetailsPage = new CaseDetailsPage();
+
+describe('Issue Decision', () => {
+	beforeEach(() => {
+		cy.login(users.appeals.caseAdmin);
+	});
+
+	it('Issue Decision Allowed ', { tags: tag.smoke }, () => {
+		let futureDate = new Date();
+		futureDate.setDate(futureDate.getDate());
```

Comment:
yes that's a good point, usually it would set the date so that it's in the future, in other tests you'd pass through + 28 after getData but I removed it so that that is sets the current date ( decision date can only be current or past date). But futureDate isn't really right. I've just amended it to ```
let date = new Date();
		date.setDate(date.getDate());
```
Do you think that's better?

Reply:
oh, if you just want the current date you don't need the `date.setDate(date.getDate());` bit, new dates are at the time they are instantiated

Reply:
<img width="450" alt="Screenshot 2024-09-11 at 11 50 14" src="https://github.com/user-attachments/assets/46c404ab-f6f0-423b-bb22-1668c0dacb78">

Reply:
Oh! yeah! Thanks, I didn't realise that, that's much simpler!